grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -122,124 +123,14 @@ class StockMovementController {
         stockMovement.documents = stockMovementService.getDocuments(stockMovement)
 
         if (stockMovement?.order) {
-            render(view: "/returns/show", model: [stockMovement: stockMovement])
+            render(view: "/returns/show", model: [stockMovement: stockMovement, currentLocation: currentLocation])
         } else {
-            render(view: "show", model: [stockMovement: stockMovement])
+            render(view: "show", model: [stockMovement: stockMovement, currentLocation: currentLocation])
         }
     }
 
     def list = {
-
-        def max = params.max ? params.max as int : 10
```

Comment:
This seems to be wrong that it's there. I think this is strictly related to the list redesign changes.

Reply:
yes that's why I drafted this PR
I believe that during reverting my commit (that I did locally) I did something wrong on the StockMovementController
everything is up to date now